notebooks/iatraoui.ipynb

```diff
@@ -301,8 +385,30 @@
     "def classify_sentinel_data(\n",
     "    rasters: xr.DataArray, classified_rgb_rows: ClassifiedSamples\n",
     ") -> ClassificationResult:\n",
-    "    ...  # TODO\n",
-    "    raise NotImplementedError\n"
+    "\n",
+    "    RF_model = RandomForestClassifier(\n",
+    "        random_state=0,\n",
+    "        n_jobs=-1,\n",
+    "        n_estimators=10,\n",
+    "        bootstrap=False,\n",
+    "        class_weight=\"balanced\",\n",
+    "    )\n",
+    "\n",
+    "    training_data = classified_rgb_rows[\"reflectance\"].values\n",
```

Comment:
You can pass the data arrays directly to the fit function (no need to access `.values`)